### PR TITLE
Added scala-cross-build

### PR DESCRIPTION
**Pull request by dvallejo**
Added scala cross build.

PR's related: #1588 #1376

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -197,43 +195,54 @@
         <easy.mock.version>2.5.2</easy.mock.version>
         <stratio.common.utils.version>0.4.0</stratio.common.utils.version>
         <stratio-platform.target>1.6</stratio-platform.target>
+        <spark.version.folder>1.5</spark.version.folder>
+        <spark.version>1.5.2</spark.version>
```
Comment (danielcsant) [inline pom.xml]: Are we using these two properties?

File: pom.xml [hunk 2]
```diff
@@ -197,43 +195,54 @@
...
+        <mongodb.spark.version>0.10.1</mongodb.spark.version>
+        <elastic.spark.version>2.2.0-beta1</elastic.spark.version>
+        <!-- Scala version and cross build properties -->
```
Comment (danielcsant) [inline pom.xml]: And these ones?

Comment (dvallejo) [inline pom.xml]: I've removed these properties.
Thanks!

**Comment by danielcsant** [comment]
+1

**Pull request closed without merging**